### PR TITLE
feature/DEV-46-bulk-delete

### PR DESCRIPTION
**Pull request by RC-3222**
Redone item deletion for skills & languages + replaced (redundant) "delete" buttons in "update" dialogs with "cancel" (as in the example)

image
image
image

image

### CODE REVIEW

File: src/components/bulk-delete-button/bulk-delete-button.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    isSelected: boolean;
+}
+
+const BulkDeleteButton = ({ item, isSelected, className, ...props }: BulkDeleteProps) => {
+    const Icon = isSelected ? DeleteFilled : DeleteOutlined;
+
+    return (
+        <Button
+            {...props}
+            type="text"
```
Comment (shev7) [inline src/components/bulk-delete-button/bulk-delete-button.tsx:20]: what is the meaning of the button with type text??

File: src/modules/cvs/components/skills/skills.tsx [hunk 2]
```diff
@@ -49,17 +50,23 @@ const Skills = ({ cvId, currentUser }: SkillProps) => {
...
                 <Button size={"large"} type="text" onClick={openAddSkillDialog}>
                     <PlusOutlined />
                     {t("skills.addSkill")}
                 </Button>
             )}
-
-            <CvSkillsContainer skills={cvData?.cv?.skills ?? []} cvId={cvId} />
+            <CvSkillsContainer
+                canEdit={canEdit}
+                skills={cvData ? cvData.cv.skills : []}
```
Comment (shev7) [inline src/modules/cvs/components/skills/skills.tsx]:
```suggestion
                skills={cvData?.cv?.skills || []}
```

File: src/modules/users/components/languages-list/item/item.tsx [hunk 3]
```diff
@@ -3,32 +3,58 @@ import classNames from "classnames";
...
+                onClick={
+                    canEdit
+                        ? () => {
+                              if (isSelected) {
+                                  bulkDeleteService.handleItemId(language.name);
+                                  return;
+                              }
+                              handleLanguageSelect(language);
+                          }
+                        : undefined
```
Comment (shev7) [inline src/modules/users/components/languages-list/item/item.tsx]: you can just disable button

**Review by shev7 — APPROVED** [review]

File: src/components/bulk-delete-button/bulk-delete-button.tsx (hunk 1 above)
Comment (RC-3222) [reply src/components/bulk-delete-button/bulk-delete-button.tsx:20]: one of the standard antd button styling variants (which looks similar to the regular text, just like in the example)
(antd button "type" is not the regular html button type)
image

File: src/modules/cvs/components/skills/skills.tsx (hunk 2 above)
Comment (RC-3222) [reply src/modules/cvs/components/skills/skills.tsx]: fixed

File: src/modules/users/components/languages-list/item/item.tsx (hunk 3 above)
Comment (RC-3222) [reply src/modules/users/components/languages-list/item/item.tsx]: fixed

**Review by savelievamareena — APPROVED** [review]

**Review by AAKlimkov — APPROVED** [review]

**Pull request merged**